po/fr.po

```diff
@@ -131,319 +144,451 @@ msgid "Highlight focused application"
 msgstr "Surligner l'application active"
 
 #: Settings.ui.h:16
-msgid "Height (px)"
-msgstr "Hauteur (px)"
+msgid "Highlight color"
+msgstr "Couleur de surlignement"
 
 #: Settings.ui.h:17
-msgid "0"
-msgstr "0"
+msgid "Highlight opacity"
+msgstr "Opacité du surlignement"
 
 #: Settings.ui.h:18
-msgid "Color - Override Theme"
-msgstr "Couleur - Remplacer le thème"
+msgid "5"
+msgstr "5"
 
 #: Settings.ui.h:19
-msgid "1 window open"
-msgstr "1 fenêtre ouverte"
+msgid "Indicator height (px)"
+msgstr "Hauteur indicateur (px)"
 
 #: Settings.ui.h:20
+msgid "0"
+msgstr "0"
+
+#: Settings.ui.h:21
+msgid "Indicator color - Override Theme"
+msgstr "Couleurde l'indicateur - Remplacer le thème"
+
+#: Settings.ui.h:22
+msgid "1 window open (or ungrouped)"
+msgstr "1 fenêtre ouverte (ou dégroupée)"
+
+#: Settings.ui.h:23
 msgid "Apply to all"
 msgstr "Appliquer à tout"
 
-#: Settings.ui.h:21
+#: Settings.ui.h:24
 msgid "2 windows open"
 msgstr "2 fenêtres ouvertes"
 
-#: Settings.ui.h:22
+#: Settings.ui.h:25
 msgid "3 windows open"
 msgstr "3 fenêtres ouvertes"
 
-#: Settings.ui.h:23
+#: Settings.ui.h:26
 msgid "4+ windows open"
 msgstr "4+ fenêtres ouvertes"
 
-#: Settings.ui.h:24
+#: Settings.ui.h:27
 msgid "Use different for unfocused"
 msgstr "Style différent pour les applications inactives"
 
-#: Settings.ui.h:25
+#: Settings.ui.h:28
+msgid "Font size (px) of the application titles (default is 14)"
+msgstr "Taille (px) du texte des titres d'application (défaut: 14)"
+
+#: Settings.ui.h:29
+msgid "Font color of the application titles"
+msgstr "Couleur du texte des titres d'application"
+
+#: Settings.ui.h:30
+msgid "Maximum width (px) of the application titles (default is 160)"
+msgstr "Longueur maximum (px) des titres d'applications (défaut: 160)"
```

Comment:
`titres d'applications` should perhaps just be
`titres d'application` as in the other translations above ?